### DISCUSSION TITLE
: variables map can't be null V16.2

### DISCUSSION THREAD

**Discussion by skumaravelan** [graphql-java/graphql-java#2299] [Q&A]
I have upgrade the GraphQL library from 11 to 16.2 and am getting graphql.AssertException: variables map can't be null error 
Is there any fix or configuration to load schema without variables.

**Comment by aleksandarsusnjar** [comment]
I experienced a similar (same?) error - not sure of your context. My context was testing with GraphiQL using HTTP GET method. It includes `variables` as a query parameter that is literally text `"null"` (excluding quotation marks), not "no-value" `null` when variables aren't specified.  That is invalid and "prints" the same. I handled this in my servlet, treating `&variables=null` as unspecified (truly null) `variables` parameter.

**Comment by andimarek** [comment]
GraphQL Java is just the execution engine which doesn't deal with Http or how to load the schema.

I expect you are using a library on top of GraphQL Java which causes this problem.

**Reply by skumaravelan** [reply]
Previous version there is no null check required and now it failed, I have added null check to move forward.
